development/build/manifest.js

```diff
@@ -22,6 +22,11 @@ function createManifestTasks({
   shouldIncludeSnow,
   entryTask,
 }) {
+  // TODO: remove this filter for firefox once MV3 is supported in it
+  const mv3BrowserPlatforms = browserPlatforms.filter(
```

Comment:
Is the intent here for this to produce an MV2 build for Firefox, even if `ENABLE_MV3` is set?

If so, there are a bunch of other build changes triggered by `ENABLE_MV3`. Right now it seems that it'd produce a mixed MV2/MV3 build for Firefox, not one or the other.

Reply:
oh, well it is handled in `createManifestV3AppInitializationBundle`, but was missed when scuttling logic was added in `setupScuttlingWrapping`

Reply:
Check out line 40 of this file; even on a Firefox build, we'll be incorporating MV3 platform modifications. There's a reference below where the manifest description will be updated to say it's MV3 as well.

Reply:
okay, yes, I see that now, thanks